Source/UI/CountrySelection/CountrySelectionViewController.swift

```diff
@@ -38,16 +38,19 @@ public final class CountrySelectionViewController: UIViewController,
             view?.overrideUserInterfaceStyle = .light
         }
         setup()
-        view.backgroundColor = CheckoutTheme.primaryBackgroundColor
+        view.backgroundColor = .white
         navigationItem.title = Constants.LocalizationKeys.BillingForm.Country.text
         // table view
         filteredCountries = countries
         tableView.delegate = self
         tableView.dataSource = self
         // search bar
         searchBar.delegate = self
-
-        tableView.backgroundColor = CheckoutTheme.primaryBackgroundColor
+        searchBar.placeholder = Constants.LocalizationKeys.CountrySelection.search
+        searchBar.barTintColor = UIColor(hex: "#F0F0F0")
+        searchBar.tintColor = UIColor(hex: "#F0F0F0")
+        searchBar.backgroundImage = UIImage()
+        tableView.backgroundColor = UIColor(hex: "#F0F0F0")
```

Comment:
I don't think we should hardcode any colours in any of the screens. Would it be reasonable for this screen to also reflect some level of customisation provided by the consumer?

Reply:
This screen will be refactored because it's old and not supporting the custom scheme like matrix.

Reply:
I had the feeling @patrick-hoban-cko and @deepesh-vasthimal-cko said everything is ready for release, we're just verifying and polishing off. This would mean a piece of work is missed?

Reply:
[Jira ticket ](https://checkout.atlassian.net/browse/PIMOB-1445?atlOrigin=eyJpIjoiYzcwMTdhNzY2NDc1NDRhYTg3NWNkYzU1MGM3ZjcyNTIiLCJwIjoiaiJ9)